### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by tapdancemobile**
Take a look at this - thanks!

### CODE REVIEW

File: index.android.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/**
```
Comment (shanejonas) [inline index.android.js]: we should probably just have one entry file

File: ios/emerald_mobile_wallet-tvOS/Info.plist [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shanejonas) [inline ios/emerald_mobile_wallet-tvOS/Info.plist]: is this tvOS file necessary?

File: ios/emerald_mobile_wallet-tvOSTests/Info.plist [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shanejonas) [inline ios/emerald_mobile_wallet-tvOSTests/Info.plist]: is this tvOS file necessary?

File: ios/emerald_mobile_wallet.xcodeproj/xcshareddata/xcschemes/emerald_mobile_wallet-tvOS.xcscheme [hunk 4]
```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shanejonas) [inline ios/emerald_mobile_wallet.xcodeproj/xcshareddata/xcschemes/emerald_mobile_wallet-tvOS.xcscheme]: more tvOS

File: rust/emerald_mobile_wallet/src/lib.rs [hunk 5]
```diff
@@ -0,0 +1,19 @@
+#[cfg(feature = "jni")]
```
Comment (shanejonas) [inline rust/emerald_mobile_wallet/src/lib.rs]: dont think we need this?

File: ios/emerald_mobile_wallet/AppDelegate.h [hunk 6]
```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```
Comment (shanejonas) [inline ios/emerald_mobile_wallet/AppDelegate.h]: BSD

File: ios/emerald_mobile_wallet/AppDelegate.m [hunk 7]
```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```
Comment (shanejonas) [inline ios/emerald_mobile_wallet/AppDelegate.m]: BSD

File: ios/emerald_mobile_wallet/main.m [hunk 8]
```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```
Comment (shanejonas) [inline ios/emerald_mobile_wallet/main.m]: BSD

File: ios/emerald_mobile_walletTests/emerald_mobile_walletTests.m [hunk 9]
```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
```
Comment (shanejonas) [inline ios/emerald_mobile_walletTests/emerald_mobile_walletTests.m]: maybe remove these tests. i dont think we will need them

File: index.android.js (hunk 1 above)
Comment (tapdancemobile) [reply index.android.js]: Got it... That's fine. I think that's just RN convention

File: ios/emerald_mobile_wallet-tvOS/Info.plist (hunk 2 above)
Comment (tapdancemobile) [reply ios/emerald_mobile_wallet-tvOS/Info.plist]: No that's just boilerplate XCode stuff

File: ios/emerald_mobile_walletTests/emerald_mobile_walletTests.m (hunk 9 above)
Comment (tapdancemobile) [reply ios/emerald_mobile_walletTests/emerald_mobile_walletTests.m]: I can remove the ios tests... you're right - we'd probably use jasmine or jest

File: ios/emerald_mobile_wallet/main.m (hunk 8 above)
Comment (tapdancemobile) [reply ios/emerald_mobile_wallet/main.m]: All iOS apps use this... Not sure we can get away with not having it

Comment (tapdancemobile) [reply ios/emerald_mobile_wallet/main.m]: This is generated by the React-Native platform

File: ios/emerald_mobile_wallet/AppDelegate.m (hunk 7 above)
Comment (tapdancemobile) [reply ios/emerald_mobile_wallet/AppDelegate.m]: All iOS apps use this... Not sure we can get away with not having it. Generated by the platform but I can rewrite with the same content?

File: ios/emerald_mobile_wallet/main.m (hunk 8 above)
Comment (tapdancemobile) [reply ios/emerald_mobile_wallet/main.m]: React-Native is not under MIT license, though they just put React under MIT

File: ios/emerald_mobile_wallet.xcodeproj/xcshareddata/xcschemes/emerald_mobile_wallet-tvOS.xcscheme (hunk 4 above)
Comment (shanejonas) [reply ios/emerald_mobile_wallet.xcodeproj/xcshareddata/xcschemes/emerald_mobile_wallet-tvOS.xcscheme]: remove tvOS file

File: index.android.js (hunk 1 above)
Comment (shanejonas) [reply index.android.js]: Lets just make one entry file that should work for both: `index.js`

File: ios/emerald_mobile_wallet.xcodeproj/xcshareddata/xcschemes/emerald_mobile_wallet-tvOS.xcscheme (hunk 4 above)
Comment (shanejonas) [reply ios/emerald_mobile_wallet.xcodeproj/xcshareddata/xcschemes/emerald_mobile_wallet-tvOS.xcscheme]: remove tvOS file

**Review by shanejonas — CHANGES_REQUESTED** [review]

**Comment by tapdancemobile** [comment]
@shanejonas - done... thanks!

**Review by shanejonas — APPROVED** [review]

**Pull request merged**